docs/development.md

```diff
@@ -105,8 +105,16 @@ The server runs on [localhost:5000](http://localhost:5000/) by default.
 
 ## Become an admin:
 
+1. On [localhost:5000](http://localhost:5000/), create an account (You can use your Google ZD account), then run this rake command:
```

Comment:
Nit: I reckon we should avoid being Zendesk-specific since this is an open source project.

```suggestion
1. On [localhost:5000](http://localhost:5000/), create an account, then run this rake command:
```